testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeDatabricksCheckpointsCompatibility.java

```diff
@@ -206,22 +208,18 @@ public void testDatabricksUsesCheckpointInterval()
         try {
             // validate that Databricks can see the checkpoint interval
             String showCreateTable;
-            if (databricksRuntimeVersion.equals(DATABRICKS_104_RUNTIME_VERSION)) {
+            if (databricksRuntimeVersion.compareTo(DATABRICKS_104_RUNTIME_VERSION) >= 0) {
```

Comment:
This is String.compareTo, a poor man's version comparison impl.

Let's have a method that hides this ugliness

perhaps `isAtLeastDatabricks104(databricksRuntimeVersion)`

the method could do some addiotiuonal validation or perhaps avoid `String.compareTo` whatsoever, since set of expected versions is very small

Reply:
> This is String.compareTo

i am wrong.

What about adding a method `isAtLeast(...)` to the version class? 
and `isOlderThan(..)` for negative check
this would be more readable than inspecting `compareTo` results.